### ISSUE TITLE
Calc: Misaligned Function wizard button next to formula bar.

### ISSUE THREAD

**Issue by thebearon** [CollaboraOnline/online#4620]
See screenshot:
![calc_misaligned_formula_bar](https://user-images.githubusercontent.com/19438782/163702307-2cfb0964-80bb-4536-a00b-f518611e2924.png)
(this is with branch `distro/collabora/co-21-11`, but it's the same with `master` branch)

This used to be fine, the alignment issue started with 8d350b28453eee6d84a8bd2b9285d5664706f342.
```
Author:     Andreas Kainz <kainz.a@gmail.com>
AuthorDate: Mon Mar 7 00:21:52 2022 +0100
Commit:     Andreas Kainz <andreas_k@abwesend.de>
CommitDate: Tue Mar 8 08:33:48 2022 +0100

    Mobile Search use var colors and is vertical aligned
```
CC @Andreas-Kainz @pedropintosilva.

**Comment by Andreas-Kainz** [comment]
Thanks will check.

**Comment by Andreas-Kainz** [comment]
And fix during the next day's.

**Comment by pedropintosilva** [comment]
Thanks

**Comment by pedropintosilva** [comment]
the merged PR is for 21.11 since Master now is pared with core 22.05 (and more changes from formula bar are coming from @eszkadev making it rendered in the client side )
